Azure/sonic-buildimage
# [Inventec][D6356] Update driver and Add new platform API implementation

**- What I did**
* Comment out management/loopback ports in BCM file
* Update TD3 M0 LED firmware loading logic
* Update D6356 Driver
* Add New Platform API implementation (Chassis, PSU)
* Add New Platform API installer for pmon and base image

**- How I did it**
* update BCM file device/inventec/x86_64-inventec_d6356-r0/INVENTEC-D6356/td3-d6356-48x25G-8x100G.config.bcm
* update and add files in device/inventec/x86_64-inventec_d6356-r0/
* update and add files in platform/broadcom/sonic-platform-modules-inventec/d6356/modules
* add files in platform/broadcom/sonic-platform-modules-inventec/d6356/sonic_platform/
* modify files in platform/broadcom/sonic-platform-modules-inventec/debian/

**- How to verify it**
* check drivers: lsmod |grep -E "^inv_"
* check platform eeprom: show platform syseeprom
* check platform PSUs:  show platform psustatus
* check interface status: show interfaces status
* check transceiver presence: show interfaces transceiver presence
* check LED status: bcmcmd 'led'

**- Description for the changelog**

* Comment out management/loopback ports in BCM file
* Update TD3 M0 LED firmware loading logic
* Update D6356 Driver
* Add New Platform API implementation (Chassis, PSU)
* Add New Platform API installer for pmon and base image

**- A picture of a cute animal (not mandatory but encouraged)**
